protostar/commands/deploy/deploy_command.py

```diff
@@ -82,7 +82,7 @@ def arguments(self) -> List[Command.Argument]:
                     # pylint: disable=line-too-long
                     "[Read more about representing Cairo data types in the CLI.](https://www.cairo-lang.org/docs/hello_starknet/more_features.html#array-arguments-in-calldata)"
                 ),
-                type="int",
+                type="felt",
                 is_array=True,
             ),
             Command.Argument(
```

Comment:
argument `salt` could also be a `felt` I think @kasperski95 correct me if I'm wrong.

Reply:
![Screenshot 2022-08-22 at 14 11 44](https://user-images.githubusercontent.com/9629577/185918280-031d4c7b-7518-4750-bfdf-0d624cff1d8d.png)